### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by slandelle**
* Fix Gatling logo (the one used was the old one)
* remove content as the documentation now lives on the official Gatling website

### CODE REVIEW

**Review by eshepelyuk — APPROVED** [review]
LGTM

File: README.adoc [hunk 1]
```diff
@@ -12,410 +12,12 @@ ifdef::env-github[]
 :warning-caption: :warning:
 endif::[]
 
-image:https://gatling.io/wp-content/uploads/2017/02/Gatling-logo.png[Gatling^, link="https://gatling.io/open-source", window="_blank"]
+image:https://gatling.io/wp-content/uploads/2019/04/logo-gatling-transparent@15x.svg[Gatling^, link="https://gatling.io", window="_blank"]
```
Comment (Isammoc) [inline README.adoc:15]: Too big?

File: README.adoc [hunk 2]
```diff
@@ -12,410 +12,12 @@ ifdef::env-github[]
...
-}
-----
-
-These options ensure that `4.0.51.Final` will be used only for `gatling`
-configurations, leaving other dependencies unchanged.
-
-== Release a new version
-
- * Push a tag `vX.Y.Z`, where `X.Y.Z` is the version to publish.
-
```
Comment (Isammoc) [inline README.adoc:413]: Perhaps add a file for dev purpose? `DEVELOPMENT.md`? `CONTRIBUTION.md`? `HOWTO.md`?

File: README.adoc (hunk 1 above)
Comment (slandelle) [reply README.adoc]: Fixed

File: README.adoc (hunk 2 above)
Comment (slandelle) [reply README.adoc:413]: Separate issue. Please take care of it.

**Review by Isammoc — APPROVED** [review]

**Pull request merged**